inbm/dispatcher-agent/fpm-template/usr/share/dispatcher-agent/manifest_schema.xsd

```diff
@@ -319,11 +319,11 @@
                             <xs:element name="add" minOccurs="1" maxOccurs="1">
                                 <xs:complexType>
                                     <xs:sequence>
-                                        <xs:element name="gpg" minOccurs="1" maxOccurs="1">
+                                        <xs:element name="gpg" minOccurs="0" maxOccurs="1">
```

Comment:
Update the schema documentation to reflect the optionality of the GPG key elements and provide examples of manifests without these elements. [medium]